tests/issues/issue-279.php

```diff
@@ -2049,8 +2049,8 @@
             'RenderingEngine_Version' => 'unknown',
             'RenderingEngine_Maker' => 'Apple Inc',
         ],
-        'lite' => true,
-        'standard' => true,
+        'lite' => false,
```

Comment:
Because I'm making the Safari version more dependent on the iOS version, if the Standard or Lite file don't contain the iOS version that I use, then they'll be detected as "Safari Generic", which is why I've changed these to `false` in some cases.